.github/workflows/post-pull-request-checks-automation.yml

```diff
@@ -65,7 +61,7 @@ jobs:
           current-workflow-id: ${{ github.run_id }}
 
       - name: Set Merge Token
-        if: steps.check-steps-should-run.outputs.should-run == 'true'
+        if: steps.check-steps-should-run.outputs.is-latest == 'true'
```

Comment:
Sorry for giving a misleading tip earlier, but these should actually check `!= 'false'`. That way, they will match if the value is `'true'` but also if it's not set because `inputs.run-only-latest-workflow` is `'false'`.